src/pages/community/handbook/contributor-journey.js

```diff
@@ -13,5 +13,5 @@ const CommunityAbout = () => {
 };
 export default CommunityAbout;
 export const Head = () => {
-  return <SEO title="Community Handbook" description="Instructions to encourage and support first-time open-source contributors." />;
-};
\ No newline at end of file
+  return <SEO title="Contributor-Journey" description="Instructions to encourage and support first-time open-source contributors." />;
```

Comment:
Why are you hyphenating?

Reply:
@leecalcote 
I saw the file name : `community/handbook/contributor-journey.js` contributor-journey.js and that's y put the title the same.
Shall I change it to `Contributor Journey`?